marketing_transactional.go

```diff
@@ -0,0 +1,50 @@
+package hubspot
+
+import "fmt"
+
+const transactionalBasePath = "transactional"
+
+// TransactionalService is an interface for the marketing/transactional service of the HubSpot API.
+// Reference: https://developers.hubspot.com/docs/api/marketing/transactional-emails
+type TransactionalService interface {
+	SendSingleEmail(props *SendSingleEmailProperties) (*SendSingleEmailResponse, error)
+}
+
+// TransactionalServiceOp provides the default implementation of TransactionService.
+type TransactionalServiceOp struct {
+	transactionalPath string
+	client            *Client
+}
+
+var _ TransactionalService = (*TransactionalServiceOp)(nil)
+
+type SendSingleEmailMessage struct {
+	To      string   `json:"to"`
+	From    string   `json:"from,omitempty"`
+	SendId  string   `json:"sendId,omitempty"`
+	ReplyTo []string `json:"replyTo,omitempty"`
+	Cc      []string `json:"cc,omitempty"`
+	Bcc     []string `json:"bcc,omitempty"`
+}
+
+type SendSingleEmailProperties struct {
+	EmailId           string                  `json:"emailId"`
```

Comment:
Should `EmailID` be a string?
The api documentation says `int32`.

Reply:
Fixed - it seems we're using int64 everywhere else in the codebase so used that.   If you prefer it to use int32, I'm fine with that too.

Where are you looking for type documentation incidentally?   The doc page I used to work against didn't have any types - it just says "a json object with these properties"  

However I dug into their official TypeScript docs and that seems to match what you're saying here.

Thanks.

Reply:
I was referring to the document below. The details of each API are described in the "Endpoints" tab.
There is no problem with the type being `int64`. thank you.

https://developers.hubspot.com/docs/api/marketing/transactional-emails#endpoint?spec=POST-/marketing/v3/transactional/single-email/send

![image](https://user-images.githubusercontent.com/49366645/216536373-8b81a930-9ddb-4a2a-9c91-fc7f0d92e96a.png)